vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py

```diff
@@ -30,11 +27,22 @@ def get_encoded_api_key(jwt_secret):
     return encoded_jwt
 
 
-@patch.object(LLMEngine, "__init__", new=logging_init_wrapper)
```

Comment:
I may be missing something, but without using the wrapper, how is the raw logging class for the api server being initialized?

Reply:
We will use it for benchmarking only. We're not enabling the stats logging by default in the main run script.